hocr-check

```diff
@@ -71,7 +71,10 @@ if len(sys.argv)<1:
 optlist,args = getopt.getopt(sys.argv[1:],"o")
 nooverlap = (assoc('-o',optlist)=='')
 if len(args)>0: stream = open(args[0])
-elif len(args)>1: raise "can only check one file at a time"
+elif len(args)>1:
+    print("Can only check one file at a time")
+    print_usage()
+    sys.exit(1)
```

Comment:
Is the `elif` case reachable at all? 

`¬(len(args)>0) && (len(args)>1)` = ??

Maybe, we should switch the order of these two cases...